### PR TITLE
New CTDMO cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New CTDMO cal csvs - autogenerated from QCTs.  I already reviewed every file & compared each to the vendor .cal file that came with the refurb documentation.  Other than the minor rounding differences that we often see (and are acceptable), these all checked out fine.  
**NOTE: 2 CTDMOs came back from vendor refurb with incorrect cal coefficients loaded on the instruments.  

- 37-10225 - correct cal coefficients loaded (taken from the vendor cal docs) - vendor .cal file is referenced as source_file in cal csv.
- 37-11603 exhibited pressure issues during initial QCT.  It was observed that the cal coefficients loaded on the instrument didn't match any recent vendor cal files; new cal coefficients were loaded onto instrument from the most recent vendor .cal file and the QCT was conducted again and the new cal csv was generated.  QCT is referenced as source_file in cal csv.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

**Comment by snwhite** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/CTDMOH/CGINS-CTDMOH-13457__20201020.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+37-13457,CC_ptca0,5.244055e+05,
+37-13457,CC_ptca1,-2.074841e-01,
+37-13457,CC_ptca2,-1.771355e-02,
+37-13457,CC_ptcb0,2.493075e+01,
+37-13457,CC_ptcb1,1.500000e-04,
+37-13457,CC_ptcb2,0.000000e+00,
+37-13457,CC_ptempa0,2.237106e+02,
+37-13457,CC_ptempa1,-1.131520e-01,
+37-13457,CC_ptempa2,5.401780e-06,
+37-13457,CC_wbotc,2.470009e-07,
```
Comment (batryn) [inline calibration/CTDMOH/CGINS-CTDMOH-13457__20201020.csv:25]: Looks good (spot check)

File: calibration/CTDMOQ/CGINS-CTDMOQ-12305__20201023.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+37-12305,CC_ptca0,5.249662e+05,
+37-12305,CC_ptca1,3.644972e+00,
+37-12305,CC_ptca2,-1.890254e-02,
+37-12305,CC_ptcb0,2.549325e+01,
+37-12305,CC_ptcb1,-5.500000e-04,
+37-12305,CC_ptcb2,0.000000e+00,
+37-12305,CC_ptempa0,-6.907313e+01,
+37-12305,CC_ptempa1,5.232980e-02,
+37-12305,CC_ptempa2,-7.587151e-07,
+37-12305,CC_wbotc,2.777271e-07,
```
Comment (batryn) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12305__20201023.csv:25]: Looks good (spot check)

File: calibration/CTDMOQ/CGINS-CTDMOQ-12306__20201023.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+37-12306,CC_ptca0,5.246988e+05,
+37-12306,CC_ptca1,6.902898e-01,
+37-12306,CC_ptca2,4.130723e-03,
+37-12306,CC_ptcb0,2.539725e+01,
+37-12306,CC_ptcb1,-1.500000e-04,
+37-12306,CC_ptcb2,0.000000e+00,
+37-12306,CC_ptempa0,-6.860888e+01,
+37-12306,CC_ptempa1,5.105424e-02,
+37-12306,CC_ptempa2,-4.669635e-07,
+37-12306,CC_wbotc,2.782739e-07,
```
Comment (batryn) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12306__20201023.csv:25]: Looks good (spot check)

File: calibration/CTDMOQ/CGINS-CTDMOQ-12307__20201029.csv [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+37-12307,CC_ptca0,5.242325e+05,
+37-12307,CC_ptca1,2.312306e+00,
+37-12307,CC_ptca2,-5.252508e-02,
+37-12307,CC_ptcb0,2.535075e+01,
+37-12307,CC_ptcb1,-2.850000e-03,
+37-12307,CC_ptcb2,0.000000e+00,
+37-12307,CC_ptempa0,-7.354585e+01,
+37-12307,CC_ptempa1,5.060820e-02,
+37-12307,CC_ptempa2,-5.463678e-07,
+37-12307,CC_wbotc,1.743527e-07,
```
Comment (batryn) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12307__20201029.csv:25]: Looks good (spot check)

File: calibration/CTDMOQ/CGINS-CTDMOQ-12607__20201023.csv [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+37-12607,CC_ptca0,5.248435e+05,
+37-12607,CC_ptca1,-4.159119e-01,
+37-12607,CC_ptca2,-1.343759e-02,
+37-12607,CC_ptcb0,2.554038e+01,
+37-12607,CC_ptcb1,2.750000e-04,
+37-12607,CC_ptcb2,0.000000e+00,
+37-12607,CC_ptempa0,-6.603543e+01,
+37-12607,CC_ptempa1,5.169225e-02,
+37-12607,CC_ptempa2,-4.792741e-07,
+37-12607,CC_wbotc,3.258783e-07,
```
Comment (batryn) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12607__20201023.csv:25]: Looks good (spot check)

File: calibration/CTDMOQ/CGINS-CTDMOQ-13379__20201020.csv [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+37-13379,CC_ptca0,5.241870e+05,
+37-13379,CC_ptca1,1.088819e+00,
+37-13379,CC_ptca2,-2.156049e-03,
+37-13379,CC_ptcb0,2.505987e+01,
+37-13379,CC_ptcb1,-2.500000e-05,
+37-13379,CC_ptcb2,0.000000e+00,
+37-13379,CC_ptempa0,1.412355e+02,
+37-13379,CC_ptempa1,-6.683798e-02,
+37-13379,CC_ptempa2,3.963357e-07,
+37-13379,CC_wbotc,1.862654e-07,
```
Comment (batryn) [inline calibration/CTDMOQ/CGINS-CTDMOQ-13379__20201020.csv:25]: Looks good (spot check)

File: calibration/CTDMOQ/CGINS-CTDMOQ-13380__20201023.csv [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+37-13380,CC_ptca0,5.250151e+05,
+37-13380,CC_ptca1,8.040849e-01,
+37-13380,CC_ptca2,2.990263e-01,
+37-13380,CC_ptcb0,2.503225e+01,
+37-13380,CC_ptcb1,5.000000e-05,
+37-13380,CC_ptcb2,0.000000e+00,
+37-13380,CC_ptempa0,1.672228e+02,
+37-13380,CC_ptempa1,-7.902516e-02,
+37-13380,CC_ptempa2,3.111110e-06,
+37-13380,CC_wbotc,0.000000e+00,
```
Comment (batryn) [inline calibration/CTDMOQ/CGINS-CTDMOQ-13380__20201023.csv:25]: Looks good (spot check)

File: calibration/CTDMOQ/CGINS-CTDMOQ-13381__20201023.csv [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+37-13381,CC_ptca0,5.230098e+05,
+37-13381,CC_ptca1,-2.073657e+00,
+37-13381,CC_ptca2,-1.157265e-02,
+37-13381,CC_ptcb0,2.509950e+01,
+37-13381,CC_ptcb1,-1.000000e-04,
+37-13381,CC_ptcb2,0.000000e+00,
+37-13381,CC_ptempa0,1.497431e+02,
+37-13381,CC_ptempa1,-7.556779e-02,
+37-13381,CC_ptempa2,2.727627e-06,
+37-13381,CC_wbotc,1.872853e-07,
```
Comment (batryn) [inline calibration/CTDMOQ/CGINS-CTDMOQ-13381__20201023.csv:25]: Looks good (spot check)

File: calibration/CTDMOR/CGINS-CTDMOR-13375__20201023.csv [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+37-13375,CC_ptca0,5.227570e+05,
+37-13375,CC_ptca1,4.012598e-01,
+37-13375,CC_ptca2,-2.855491e-02,
+37-13375,CC_ptcb0,2.497814e+01,
+37-13375,CC_ptcb1,-1.371571e-03,
+37-13375,CC_ptcb2,0.000000e+00,
+37-13375,CC_ptempa0,-6.950681e+01,
+37-13375,CC_ptempa1,5.087449e-02,
+37-13375,CC_ptempa2,-4.951714e-07,
+37-13375,CC_wbotc,3.118929e-07,
```
Comment (batryn) [inline calibration/CTDMOR/CGINS-CTDMOR-13375__20201023.csv:25]: Looks good (spot check)

File: calibration/CTDMOR/CGINS-CTDMOR-13376__20201127.csv [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+37-13376,CC_ptca0,5.246476e+05,
+37-13376,CC_ptca1,2.392989e+00,
+37-13376,CC_ptca2,-6.348036e-02,
+37-13376,CC_ptcb0,2.500613e+01,
+37-13376,CC_ptcb1,-1.750000e-04,
+37-13376,CC_ptcb2,0.000000e+00,
+37-13376,CC_ptempa0,2.800679e+02,
+37-13376,CC_ptempa1,-1.386694e-01,
+37-13376,CC_ptempa2,8.108790e-06,
+37-13376,CC_wbotc,1.237838e-07,
```
Comment (batryn) [inline calibration/CTDMOR/CGINS-CTDMOR-13376__20201127.csv:25]: Looks good (spot check)

File: calibration/CTDMOR/CGINS-CTDMOR-13377__20201024.csv [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+37-13377,CC_ptca0,5.245750e+05,
+37-13377,CC_ptca1,7.983379e-01,
+37-13377,CC_ptca2,1.906003e-03,
+37-13377,CC_ptcb0,2.498750e+01,
+37-13377,CC_ptcb1,-3.000000e-04,
+37-13377,CC_ptcb2,0.000000e+00,
+37-13377,CC_ptempa0,1.986387e+02,
+37-13377,CC_ptempa1,-8.864399e-02,
+37-13377,CC_ptempa2,-9.242481e-07,
+37-13377,CC_wbotc,2.811162e-07,
```
Comment (batryn) [inline calibration/CTDMOR/CGINS-CTDMOR-13377__20201024.csv:25]: Looks good (spot check)

**Review by batryn — APPROVED** [review]
Looks good. All files checked or spot checked

**Review by cdobs — APPROVED** [review]
Proxy approval for @rtravis-whoi

**Pull request merged**